app/pb_kits/playbook/pb_kit/pb_date_time.rb

```diff
@@ -36,7 +36,7 @@ def to_year
       end
 
       def to_month
-        @value.strftime("%^b")
+        @value.strftime("%b")
```

Comment:
@christinaatai this is cool but it's causing a test to break in rspec.

Reply:
@coleerikson Let me look into it